src/apis/Login.ts

```diff
@@ -0,0 +1,97 @@
+import {login as kakaoLogin} from '@react-native-seoul/kakao-login';
+import {post} from './methods';
+import {NativeModules, Platform} from 'react-native';
+import {NaverLoginInitParams} from '@/types/Login';
+import {NaverLoginResponse} from '@/types/Login';
+
+//TODO: 토큰 스토리지 저장 flow
+
+// 네이버 로그인 관련 설정
+const {RNNaverLogin} = NativeModules;
+const initializeNaver = ({
+  appName,
+  consumerKey,
+  consumerSecret,
+  disableNaverAppAuthIOS = false,
+  serviceUrlSchemeIOS = '',
+}: NaverLoginInitParams) => {
+  if (Platform.OS === 'ios') {
+    if (!serviceUrlSchemeIOS) {
+      console.log('serviceUrlSchemeIOS is missing in iOS initialize.');
+      return;
+    }
+    RNNaverLogin.initialize(
+      serviceUrlSchemeIOS,
+      consumerKey,
+      consumerSecret,
+      appName,
+      disableNaverAppAuthIOS,
+    );
+  } else if (Platform.OS === 'android') {
+    RNNaverLogin.initialize(consumerKey, consumerSecret, appName);
+  }
+};
+
+const naverLogin = (): Promise<NaverLoginResponse> => {
+  return RNNaverLogin.login();
+};
+
+// TODO: 이 객체 입니다
+const naverLoginParams = {
+  appName: '이거 객체 지우시고',
+  consumerKey: '슬랙 객체 복붙',
+  consumerSecret: '하시면',
+  serviceUrlSchemeIOS: '된답니다',
+  disableNaverAppAuthIOS: false,
+};
+// 네이버 로그인 함수
+export const signInWithNaver = async () => {
+  initializeNaver(naverLoginParams);
```

Comment:
```suggestion
export const signInWithNaver = async (): Promise<boolean> => {
  initializeNaver(naverLoginParams);
```

클라이언트에서 성공 실패를 알 수 있게 boolean 값을 리턴해줄 필요가 있을 것 같습니다. response 값의 return값을 리턴하는 방법이면 될 것같네요